lib/Conversion/TorchToMhlo/Basic.cpp

```diff
@@ -843,10 +843,19 @@ LogicalResult ConvertAtenOp<AtenBatchNormOp>::matchAndRewrite(
     return success();
   } else {
     Type outputTy = getTypeConverter()->convertType(op.getType());
-    rewriter.replaceOpWithNewOp<mhlo::BatchNormInferenceOp>(
-        op, outputTy, input, weight, bias, runningMean, runningVar,
-        rewriter.getFloatAttr(inputTy.getElementType(), eps),
-        rewriter.getI64IntegerAttr(1));
+    SmallVector<int64_t, 4> castShape{inputTy.getShape().begin(),
+                                      inputTy.getShape().end()};
+    castShape[1] = weightTy.getShape()[0];
+    auto castTy = RankedTensorType::get(castShape, inputTy.getElementType());
+    // feature counts must match among operands of mhlo::BatchNormInferenceOp
```

Comment:
Nit: capitalize and use punctuation

https://llvm.org/docs/CodingStandards.html#commenting